iphone/Maps/UI/PlacePage/PlacePageLayout/Layouts/PlacePageCommonLayout.swift

```diff
@@ -184,8 +184,9 @@ extension PlacePageCommonLayout {
       isBookmark = true
     }
     if let title = placePageData.previewData.title {
-      header?.setTitle(title)
-      placePageNavigationViewController.setTitle(title)
+      let secondaryTitle = placePageData.previewData.secondaryTitle
+      header?.setTitle(title, secondaryTitle: secondaryTitle)
+      placePageNavigationViewController.setTitle(title, secondaryTitle: secondaryTitle)
```

Comment:
Interesting, why titles are set in two different places?

Reply:
`placePageNavigationController` traced to only one usage: https://github.com/organicmaps/organicmaps/blob/e7dc193c23b10c7950f8cede31091a8e0890748f/iphone/Maps/UI/PlacePage/PlacePageViewController.swift#L297

Navigation bar should appear on scroll. But I did not find scenario to trigger `scrollViewDidScroll` here, it requires more research.